Speckle_Engine/Convert/HashTest.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace BH.Engine.Speckle
+{
+    public static partial class Query
+    {
+        public static int GetHashCode(this BH.oM.Base.BHoMObject obj)
```

Comment:
I would remove this file if that is not relevant for the pr, to keep the project clean

Reply:
Sorry, forgot to delete this